### PR TITLE
Add documentation for sysctl gatherer

### PR DESCRIPTION
**Pull request by rtorrero**
This PR adds the documentation for the sysctl gatherer added in https://github.com/trento-project/agent/pull/268

### CODE REVIEW

File: guides/gatherers.md [hunk 1]
```diff
@@ -750,6 +751,43 @@ Example output (in Rhai):
 
 For extra information refer to [trento-project/agent/../gatherers/sbddump_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbddump_test.go)
 
+### syssysctltemd
```
Comment (nelsonkopliku) [inline guides/gatherers.md:754]: did you mean `sysctl`?

Comment (rtorrero) [reply guides/gatherers.md:754]: I meant `sysctl`, but my fingers apparently disagree :sweat_smile:  thanks!

**Review by nelsonkopliku — APPROVED** [review]
LGTM

**Review by arbulu89 — APPROVED** [review]
Looks good @rtorrero 
Fix the conflicts and ready to merge

**Pull request merged**